File: artifacts/snudown/decompose/src/link_ref.c
struct link_ref
{
  unsigned int id;
  struct buf *link;
  struct buf *label;
  struct buf *title;
  struct link_ref *next;
}
/*** DEPENDENCIES:
struct buf
{
  uint8_t *data;
  size_t size;
  size_t asize;
  size_t unit;
}
----------------------------
***/

File: artifacts/snudown/translate_ablation/src/link_ref.rs
use crate::*;

#[derive(Clone)]
pub struct LinkRef {
    pub id: u32,
    pub link: Option<Box<Buf>>,
    pub label: Option<Box<Buf>>,
    pub title: Option<Box<Buf>>,
    pub next: Option<Box<LinkRef>>,
}

